docs/source/user_guide/index.rst

```diff
@@ -1,13 +1,14 @@
 .. _user_guide:
 
-Creating an Image Processing Pipeline
-=====================================
+User Guide
+==========
 
 Welcome to the user guide for building an image processing pipeline using starfish! This tutorial
 will cover all the steps necessary for going from raw images to a single cell gene expression
-matrix. If you are wondering what is starfish and only have a few minutes to try it out, check
-out a pre-built pipeline by following the :ref:`quick start tutorial <quick start>`. If you are
-ready to learn how to build your own image processing pipeline using starfish then read on!
+matrix. If you are wondering what is starfish, check out :ref:`The Introduction
+<introduction>`. If you only have a few minutes to try out starfish, check out a pre-built
```

Comment:
We no longer have an introduction page, but maybe it could link to the landing page? Interestingly, this links to trackpy's introduction :astonished:

Reply:
lol

Reply:
I just killed the sentence. Also the dangling thing about the data model.